example/samples/install_esx_payload_full.json

```diff
@@ -57,6 +57,10 @@
                 }
             ],
             "installDisk": "t10.ATA_____SATADOM2DSH_TYPE_C_3SE___________________20151007AA8511725043",
+            "gdbPort": "default",
+            "logPort": "default",
+            "comport": "com1",
+            "debugLogToSerial": "1",
```

Comment:
the literal "1" is not friendly for user to understand its meaning, I see each literal have good description in https://github.com/RackHD/on-tasks/blob/master/lib/task-data/schemas/install-esxi.json#L78, so I suggest enhancing it to some meaningful word(s)